test/regression/cases/ddot_metrics/experiment.yaml

```diff
@@ -4,12 +4,11 @@ erratic: false
 target:
   name: datadog-agent
   cpu_allotment: 4
-  memory_allotment: 2GiB
+  memory_allotment: 7.5 GiB
 
   environment:
     DD_API_KEY: a0000001
```

Comment:
I believe the parsing bug requiring an `a` prefix has now been fixed

Reply:
Ah maybe. No harm to this though I do believe.